nuxeo-core/nuxeo-core-storage/src/main/java/org/nuxeo/ecm/core/storage/FulltextExtractorWork.java

```diff
@@ -209,4 +214,17 @@ protected BlobHolder convert(BlobHolder blobHolder) throws ConversionException {
         return conversionService.convert(ANY2TEXT, blobHolder, null);
     }
 
+    protected static class MimetypeAndText implements Serializable {
```

Comment:
Can you please use `StringBlob` to hold the mime type and text rather than this specialized class?